src/server.c

```diff
@@ -1685,9 +1685,9 @@ int clientsCronResizeQueryBuffer(client *c) {
     time_t idletime = server.unixtime - c->lastinteraction;
 
     /* There are two conditions to resize the query buffer:
-     * 1) Query buffer is > PROTO_RESIZE_THRESHOLD and too big for latest peak.
-     * 2) Query buffer is > PROTO_RESIZE_THRESHOLD and client is idle. */
-    if (querybuf_size > PROTO_RESIZE_THRESHOLD &&
+     * 1) Query buffer is > BIG_ARG and too big for latest peak.
+     * 2) Query buffer is > BIG_ARG and client is idle. */
+    if (querybuf_size > PROTO_MBULK_BIG_ARG &&
```

Comment:
i still think we want a separate / explicit `PROTO_RESIZE_THRESHOLD` define in server.h
this mechanism has nothing to do with the `BIG_ARG` mechanism (that's used to avoid memcpy).
i just said i'd rather avoid increasing it from 32k to 64k, but still think the separate constant is good.